### PR TITLE
Bring Shell Code Into Shellcheck Linter Compliance

### PR DESCRIPTION
**Pull request by jeremiahpslewis**
Fixes a first chunk of https://github.com/hzi-braunschweig/SORMAS-Project/issues/4697

Collaboration with @EdouM

### CODE REVIEW

**Comment by jlewis91** [comment]
Hey @JonasCir, would love to get your feedback on the changes to ``server-update.sh`` before I go forward here (there are 11 files in total ripe for linting).

**Comment by jeremiahpslewis** [comment]
@JonasCir Thanks for the second review / merge on the markdown docs. Just reconsidered what is probably the best approach here, think that a few more, smaller PRs is safer, more digestible, than one PR which attempts to fix all bash lint recommendations. So with that said, this one should be good to go (absolutely no rush and I'll wait to open up the next PR until this one is ready & merged). :)

File: .github/workflows/linter.yml [hunk 1]
```diff
@@ -46,12 +46,11 @@ jobs:
       # Run Linter against code base #
       ################################
       - name: Lint Code Base
-        uses: github/super-linter@v3.15.1
```
Comment (JonasCir) [inline .github/workflows/linter.yml:49]: I think it it possible to do something similar to `@v3.*` here? I don't fear any trouble from always loading the last stable version.

**Comment by JonasCir** [comment]
@jeremiahpslewis Thank you for pushing this! I totally agree that it is better to fix the errors step by step. At a first look changes look good but I need to delay the full review!

**Comment by jeremiahpslewis** [comment]
No worries, take your time!

**Pull request closed without merging**